jdaviz/configs/default/plugins/plot_options/tests/test_plot_options.py

```diff
@@ -76,6 +76,7 @@ def test_stretch_histogram(cubeviz_helper, spectrum1d_cube_with_uncerts):
     po.plugin_opened = True
 
     assert po.stretch_histogram is not None
+    assert "colorbar" in po.stretch_histogram._marks
```

Comment:
should we ensure it is also populated with data?

Reply:
You caught me!

Reply:
Test modified.